terratest/run-tests-from-cmdline.md

```diff
@@ -0,0 +1,30 @@
+# Running tests from the command line
+
+The test can also be run from the command line which avoids needing to have a license needed to use the IntelliJ plugin.
+
+## Prerequisites
+- A local clone of this repo
+- Install an appropriate version go (v1.13+)
+- Install dep (```curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh```)
```

Comment:
Is this the officially recommended way to install this? This type of thing where you're executing the results of a `curl` is kinda sketch.